### PR TITLE
import tsv with large # of rows in python3

### PR DESCRIPTION
**Pull request by schaluva**

### CODE REVIEW

File: scripts/import_large_tsv/import_large_tsv_python3.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    # open input tsv file
+    with open(tsv, "r") as tsvfile:
+        headers = tsvfile.readline()
+        tsv_data = headers
+
+        # for each line in tsv line, add to header. chunk by 5000 lines.
+        for i, line in enumerate(tsvfile):
+            tsv_data += line
+
+            if i > 0 and i % 5000 == 0:
```
Comment (mmorgantaylor) [inline scripts/import_large_tsv/import_large_tsv_python3.py:26]: can you add a comment to explain this logic? it's not transparent to me how this is working

Comment (schaluva) [reply scripts/import_large_tsv/import_large_tsv_python3.py:26]: Can you see if these comments make more sense?

Comment (mmorgantaylor) [reply scripts/import_large_tsv/import_large_tsv_python3.py:26]: so much better, nice work!

**Review by mmorgantaylor — APPROVED** [review]
looks good, assuming you've tested this with a large file in a workspace!

**Pull request merged**